client/app/views/recipient/tips.html

```diff
@@ -175,11 +180,12 @@
                 <span data-ng-show="sortReverse" class="fa-solid fa-caret-up"></span>
               </span>
             </span>
+            <span  ng-dropdown-multiselect="" options="dropdownScoreData" selected-model="dropdownScoreModel" events="on_changed"  extra-settings="dropdownSettings"></span>
           </th>
         </tr>
       </thead>
       <tbody id="tipListTableBody">
-        <tr id="tip-{{$index}}"
+        <tr id="tip-{{$index}}"`
```

Comment:
why this character is added? seems a mistake.

Reply:
still to be fixed.

I'm going to fix these myself while integrating the pull request.